dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestExceptionHandlerActorTest.java

```diff
@@ -0,0 +1,65 @@
+package io.github.ztkmkoo.dss.core.actor.rest;
+
+import akka.actor.testkit.typed.javadsl.TestProbe;
+import akka.actor.typed.ActorRef;
+import io.github.ztkmkoo.dss.core.actor.AbstractDssActorTest;
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceRequest;
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceResponse;
+import io.github.ztkmkoo.dss.core.actor.rest.service.DssRestActorJsonService;
+import io.github.ztkmkoo.dss.core.actor.rest.service.DssRestActorService;
+import io.github.ztkmkoo.dss.core.exception.DssRestServiceMappingException;
+import io.github.ztkmkoo.dss.core.exception.handler.DssRestExceptionHandlerResolver;
+import io.github.ztkmkoo.dss.core.message.rest.*;
+import io.github.ztkmkoo.dss.core.network.rest.enumeration.DssRestMethodType;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DssRestExceptionHandlerActorTest extends AbstractDssActorTest {
+
+    private static final TestProbe<DssRestChannelHandlerCommand> probe = testKit.createTestProbe();
+    private static final ActorRef<DssRestExceptionHandlerCommand> restExceptionHandlerActorRef = testKit.spawn(DssRestExceptionHandlerActor.create(DssRestExceptionHandlerResolver.getInstance().getExceptionHandlerMap()), "rest-master");
+    private static final DssRestServiceActorCommandRequest commandRequest = new DssRestServiceActorCommandRequest(DssRestServiceActorCommandRequest
+            .builder()
+            .channelId("test")
+            .sender(probe.ref())
+            .methodType(DssRestMethodType.GET)
+            .path("/test")
+            .build());
+
+    @Test
+    void handleException() {
+        restExceptionHandlerActorRef.tell(DssRestExceptionHandlerCommandRequest
+                .builder()
+                .service(testService())
+                .request(commandRequest)
+                .exception(new Exception())
+                .build());
+
+        probe.expectMessageClass(DssRestChannelHandlerCommandResponse.class);
```

Comment:
Could you test more bout the response not only the expected class.
I think we can expect the status code in the response at leat.

Reply:
Ok. I don't think about it.

I will add status code and channelid assertEquals.
```java
assertEquals(HttpResponseStatus.BAD_REQUEST.code(), response.getStatus().intValue());
assertEquals("test", response.getChannelId());
```